doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,71 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
```

Comment:
Can we add two sections to this document: (1) Explain PodSlice, single-host, multi-host, etc., and (2) Ray/KubeRay versions for multi-host TPU support and autoscaling?

Reply:
c9f87b8 adds a section on autoscaling and [e4b38fc](https://github.com/ray-project/ray/pull/45924/commits/e4b38fc0a89cfc9425a707bed0f32f0a53da9171) adds a section called "About TPUs" which outlines terminology such as Pod slice and single/multi host.